api/backend.go

```diff
@@ -77,7 +77,6 @@ type Backend interface {
 	GetBlockReceipts(ctx context.Context, blockHash common.Hash) types.Receipts
 	GetTxLookupInfoAndReceipt(ctx context.Context, hash common.Hash) (*types.Transaction, common.Hash, uint64, uint64, *types.Receipt)
 	GetTxAndLookupInfo(hash common.Hash) (*types.Transaction, common.Hash, uint64, uint64)
-	GetTd(blockHash common.Hash) *big.Int
```

Comment:
Nice. Can we remove it from node/cn/api_backend.go as well?

Reply:
Thank you https://github.com/kaiachain/kaia/pull/134/commits/d81f8ea344984971c9aa56897913c95a89f06ede